src/cascadia/TerminalApp/Pane.cpp

```diff
@@ -1711,7 +1719,7 @@ void Pane::_SetupChildCloseHandlers()
 IPaneContent Pane::_takePaneContent()
 {
     _closeRequestedRevoker.revoke();
-    return std::move(_content);
+    return _content ? std::move(_content) : nullptr;

```

Comment:
I think the old code should work even if `_content == nullptr`. It's like moving an empty `vector<>`.